evennia/plugin.py

```diff
@@ -0,0 +1,217 @@
+"""
```

Comment:
Not sure about the position of this module - but it's hard to claim it belongs in utils/ or server/ per-se ... Probably would also make more sense to name it `plugins.py` since it handles all plugins.

Reply:
I originally had it inside of utils, but this caused some strange behavior due to evennia/utils/__init__.py doing some kind of import that appeared to interact badly with the evennia_launcher - side of things. Although, that might have been my earlier attempt using evennia.utils.utils.class_from_module which it turned out is not usable until Django has finished loading.